src/components/DocumentationTopic/Summary/TechnologyList.vue

```diff
@@ -69,7 +69,16 @@ export default {
 </script>
 
 <style scoped lang="scss">
+@import 'docc-render/styles/_core.scss';
+
+.technologies, .technology-list {
+  display: flex;
+}
+
 .name {
+  font-size: rem(14px);
```

Comment:
Can we use `@include font-styles(label-reduced);` here too?

Reply:
Deleting this, since we applied the change in `Summary`